src/test/java/seedu/address/logic/parser/EditCommandMacroParserTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.EditCommand;
+import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
+import seedu.address.model.person.lead.Lead;
+import seedu.address.model.person.lead.LeadType;
+
+public class EditCommandMacroParserTest {
```

Comment:
We should add tests for each field in the future.

Reply:
They belong in the same testing domain, no need.

But we should add tests to make sure it can parse the right user input to make the proper EditCommandMacrioParser object